packages/babel-plugin-jsx-pragmatic/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Could u try removing those per-directory tsconfigs? Do they change anything in how things work now? In Changesets, this is all we have: https://github.com/atlassian/changesets/blob/c426035565cfac518238c8bf32f3c496c66c0657/tsconfig.json